.ci/ci_functions.sh

```diff
@@ -0,0 +1,46 @@
+function configure () {
+    rm -rf $NFS_WORKSPACE
```

Comment:
Please verify all the shell scripts with shellcheck and fix the issues (e.g. it's integrated into PyCharm IDE)

Reply:
Done